apps/cli/src/commands/run.ts

```diff
@@ -0,0 +1,64 @@
+import { Command } from "@oclif/core";
+import path from "path";
+import fs from "fs-extra";
+import { execa } from "execa";
+
+export default class Run extends Command {
+    static summary = "Run application node.";
+
+    static description = "Run a local cartesi node for the application.";
+
+    static examples = ["<%= config.bin %> <%= command.id %>"];
+
+    public async run(): Promise<void> {
+        const snapshot = path.join(".sunodo", "image");
+
+        // check if snapshot exists
+        if (!fs.existsSync(snapshot) || !fs.statSync(snapshot).isDirectory()) {
+            throw new Error(
+                "Cartesi machine snapshot not found, run 'sunodo build'"
+            );
+        }
+
+        // read hash of the cartesi machine snapshot
+        const hash = fs
+            .readFileSync(path.join(snapshot, "hash"))
+            .toString("hex");
+        const projectName = hash.substring(0, 8);
+
+        // XXX: how to know the version from the machine?
+        const version = "devel";
+
+        // resolve compose file location based on version
+        const composeFile = path.join(
+            path.dirname(new URL(import.meta.url).pathname),
+            "..",
+            "node",
+            version,
+            "docker-compose.yml"
+        );
+
+        const args = [
+            "compose",
+            "--file",
+            composeFile,
+            "--project-directory",
+            ".",
+            "--project-name",
+            projectName,
+        ];
+
+        try {
+            // clean host compose environment, deployments host volume
+            await fs.emptyDir(path.join(".sunodo", "deployments"));
+
+            // run compose environment
+            await execa("docker", [...args, "up"], { stdio: "inherit" });
```

Comment:
Could we run `docker compose up --detach --wait` and run `docker compose logs --follow` later.

The idea is to remove the `anvil` noisy logs, and maybe add a `sunodo run run --verbose` argument to add the avil logs.

What do you think?